scheme/arm-cca/evidence_handler.go

```diff
@@ -117,84 +115,48 @@ func (s EvidenceHandler) AppraiseEvidence(
 	ec *proto.EvidenceContext, endorsementsStrings []string,
 ) (*ear.AttestationResult, error) {
 	var endorsements []handler.Endorsement // nolint:prealloc
-
-	result := handler.CreateAttestationResult(SchemeName)
-
-	for i, e := range endorsementsStrings {
-		var endorsement handler.Endorsement
-
-		if err := json.Unmarshal([]byte(e), &endorsement); err != nil {
-			return nil, fmt.Errorf("could not decode endorsement at index %d: %w", i, err)
-		}
-
-		endorsements = append(endorsements, endorsement)
-	}
-
-	err := populateAttestationResult(result, ec.Evidence.AsMap(), endorsements)
-
-	// TO DO: Handle Unprocessed evidence when new Attestation Result interface
-	// is ready. Please see issue #105
-	return result, err
-}
-
-func populateAttestationResult(
-	result *ear.AttestationResult,
-	evidence map[string]interface{},
-	endorsements []handler.Endorsement,
-) error {
-	claims, err := common.MapToClaims(evidence["platform"].(map[string]interface{}))
+	var err error
+	var status ear.TrustTier
+	endorsements, err = getEndorsementsFromString(endorsementsStrings)
```

Comment:
Are the variable declarations for `endorsements` and `err` really needed?

Why not instead:

```suggestion
	endorsements, err := getEndorsementsFromString(endorsementsStrings)
```